packages/cascara/src/atoms/DatePicker/DatePicker.js

```diff
@@ -2,13 +2,15 @@ import React, { useCallback } from 'react';
 import 'antd/dist/antd.css';
 import { DatePicker as AntDatePicker } from 'antd';
 import pt from '@espressive/prop-types';
+import PICKER_TYPE from './_globals';
 
 const propTypes = {
   /** Function that will hold the result-date selected */
-  onChange: pt.function,
+  onChange: pt.func,
   /** String that will define the type of calendar to present date|week|month|quarter|year */
-  picker: pt.string,
+  picker: pt.oneOf([...Object.keys(PICKER_TYPE), null]),
```

Comment:
`picker` is already an optional property, I think `null` is not needed here

Reply:
It is. If I don't include it, I have a console error. I think that is part of the `oneOf`